apps/chat/src/components/Chat/Publish/PublishWizard.tsx

```diff
@@ -217,7 +217,7 @@ export function PublishModal({
             }))
           : otherTargetAudienceFilters;
       const folderRegExp = new RegExp(
-        entity.folderId.split('/').slice(2).join('/'),
+        entity.folderId.split('/').slice(1).join('/'),
```

Comment:
Why do we remove 1 first path component?
![image](https://github.com/epam/ai-dial-chat/assets/20163971/8c46b737-41ef-4aff-9925-b79c89971258)
And how it's related to dots in the end of name?

Reply:
I corrected the mistake, it was in a different part